src/database/models/accomodation.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+/////////////////////////////////
+
+'use strict';
+import { Model } from 'sequelize';
+export default (sequelize, DataTypes) => {
+  class accomodation extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate({ Room, Location }) {
+      this.hasMany(Room, { foreignKey: 'accomodationId', as: 'rooms' });
+      this.belongsTo(Location, { foreignKey: 'locationId', as: 'locations' });
```

Comment:
this association with location isn't working properly. user should add location which is available in location table and deny when it is not available
<img width="354" alt="Screen Shot 2022-06-28 at 15 28 32" src="https://user-images.githubusercontent.com/92625609/176190966-d4be3be4-9a9f-4a0f-a3a5-2b395d6c4918.png">